### PR TITLE
Deep Factor [WIP]

### PR DESCRIPTION
**Pull request by dcmaddix**
Minor changes to pull request https://github.com/awslabs/gluon-ts/pull/259
Paper: https://arxiv.org/pdf/1905.12417.pdf

- Updated example code to get metrics approx 0.26 on electricity
- Corrected file paths
- Corrected minor documentation shapes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-271/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-271/1/index.html

**Comment by szha** [comment]
Job PR-271/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-271/2/index.html

**Comment by lostella** [comment]
OK looks like GP tests are being shaky

File: src/gluonts/model/deep_factor/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
```
Comment (lostella) [inline src/gluonts/model/deep_factor/__init__.py]: We may want to do something similar to DeepAR here: https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/model/deepar/__init__.py

**Comment by szha** [comment]
Job PR-271/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-271/3/index.html

File: src/gluonts/model/deep_factor/__init__.py (hunk 1 above)
Comment (dcmaddix) [reply src/gluonts/model/deep_factor/__init__.py]: added in next commit

**Comment by szha** [comment]
Job PR-271/4 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-271/4/index.html

**Pull request merged**